agent/structs/acl.go

```diff
@@ -45,41 +45,68 @@ const (
 
 	// This policy gives unlimited access to everything. Users
 	// may rename if desired but cannot delete or modify the rules.
-	ACLPolicyGlobalManagementID = "00000000-0000-0000-0000-000000000001"
-	ACLPolicyGlobalManagement   = `
-acl = "write"
+	ACLPolicyGlobalManagementID   = "00000000-0000-0000-0000-000000000001"
+	ACLPolicyGlobalManagementName = "global-management"
+	ACLPolicyGlobalManagementDesc = "Builtin Policy that grants unlimited access"
+
+	ACLPolicyGlobalReadOnlyID   = "00000000-0000-0000-0000-000000000002"
```

Comment:
Yeah, I had this same thought about the ID, but there was a lot of discussion around this already. @lornasong can provide more context.

Reply:
Hey @cthain!

Re: using `00000000-0000-0000-0000-000000000002` for the policy-id

The thinking behind using `02` (instead of `03` or something) is that when listing the policies, it would look sequential since we already have (and only have) `01` for policies. It could be confusing to users to see a policy with `01` and then another with `03`. Here's a [link to the original discussion in an ADR](https://docs.google.com/document/d/1tJt9LQKCMLKP1hG8AtO0WhddemHq40Ya4AZ8y6hSGd0/edit?disco=AAAAyzSe_0I). Let me also link it in the RFC.  Open to your input though!

Re: moving the code for builtin ACLs to `consul/acl` package

I'm not familiar enough with the codebase and personally open to your suggestion. Should we go ahead and move it?

Reply:
Awesome, thanks for linking to the ADR, that's really helpful context! I'm fine with `00..02` as the policy ID.

w.r.t. moving this to the `consul/acl` package, it is a bit of an invasive change for this PR so let's keep things where they are for now and we (maybe the ZTS team?) can revisit this later.